src/chart.js

```diff
@@ -30,6 +31,9 @@ Chart.helpers.each(scales, function(scale, type) {
 	Chart.scaleService.registerScaleType(type, scale, scale._defaults);
 });
 
+// Built-in adapters (loaded for side effects)
```

Comment:
what side effects?

Reply:
https://github.com/chartjs/Chart.js/pull/5960/files#diff-d974ff8f4395b48770b414ec0a9a6ec9R27

Reply:
Why is it necessary to load it separately here? Won't it happen when required in the time scale?

Reply:
> Won't it happen when required in the time scale?

No, it won't. The time scale imports the core adapter (`./core/core.adapters.js`), which one initially provides an abstract implementation. Here we are loading the built-in adapter (`moment`), which one will override the abstract implementation. It's done here and separately to behave as if moment was an external adapter.